gh-badges/lib/index.js

```diff
@@ -27,10 +50,22 @@ class BadgeFactory {
    * @see https://github.com/badges/shields/tree/master/gh-badges/README.md
    */
   create(format) {
+    try {
+      schema.check(format)
+    } catch (e) {
+      if (e instanceof c.ContractError) {
+        const wrappedError = new ValidationError()
+        Object.assign(wrappedError, e)
+        throw wrappedError
```

Comment:
Instead of directly throwing `ContractError` here, I've wrapped it in a `ValidationError`.

My rationale here is that if we wanted to change the implementation of the validation we could do that without it being a breaking change, but I'm in 2 minds about whether its actually a useful abstraction if we just expose all the fields of the `ContractError` anyway. Thoughts?

Reply:
I've no strong feelings either way.